test/integration/includes.cc

```diff
@@ -209,6 +209,19 @@ TEST(IncludesTest, Includes)
   EXPECT_EQ("", model1->PoseRelativeTo());
   ASSERT_NE(nullptr, model1->Element());
   EXPECT_TRUE(model1->Element()->HasElement("plugin"));
+
+  const sdf::Model *model2 = world->ModelByIndex(2);
+  ASSERT_NE(nullptr, model2);
+  EXPECT_EQ("test_model_with_file", model2->Name());
+  EXPECT_FALSE(model2->Static());
+  EXPECT_EQ(1u, model2->LinkCount());
+  ASSERT_FALSE(nullptr == model2->LinkByIndex(0));
+  ASSERT_FALSE(nullptr == model2->LinkByName("link"));
+  EXPECT_EQ(model2->LinkByName("link")->Name(), model2->LinkByIndex(0)->Name());
+  EXPECT_TRUE(nullptr == model2->LinkByIndex(1));
+  EXPECT_TRUE(model2->LinkNameExists("link"));
+  EXPECT_FALSE(model2->LinkNameExists("coconut"));
+  EXPECT_EQ("1.6", model2->Element()->OriginalVersion());
```

Comment:
Does the version number of the includes.sdf need to be bumped to 1.8?

Reply:
Bumped to 1.8 in 5b80879.